taskana-adapter-camunda-wildfly-example/src/test/resources/int-test-standalone.xml

```diff
@@ -437,15 +431,24 @@
           <permission class-name="org.wildfly.security.auth.permission.LoginPermission"/>
         </permission-set>
         <permission-set name="default-permissions">
-          <permission class-name="org.wildfly.extension.batch.jberet.deployment.BatchPermission"
-                      module="org.wildfly.extension.batch.jberet" target-name="*"/>
           <permission class-name="org.wildfly.transaction.client.RemoteTransactionPermission"
                       module="org.wildfly.transaction.client"/>
           <permission class-name="org.jboss.ejb.client.RemoteEJBPermission"
                       module="org.jboss.ejb-client"/>
+          <permission class-name="org.wildfly.extension.batch.jberet.deployment.BatchPermission"
+                      module="org.wildfly.extension.batch.jberet" target-name="*"/>
         </permission-set>
       </permission-sets>
       <http>
+        <http-authentication-factory name="application-http-authentication"
```

Comment:
Why did you add this new http-authenrication-factory and the new security-domain?

Reply:
I copied it from original config file